sjb/config/common/test_cases/origin_minimal.yml

```diff
@@ -94,6 +94,11 @@ post_actions:
     script: |-
       trap 'exit 0' EXIT
       if [[ -n "${JOB_SPEC:-}" ]]; then
+        # Set the original GCE SA and project to openshift-gce-devel that can upload results to GCS bucket
+        # in case different SA was used to provision/teardown GCE cluster
+        gcloud auth activate-service-account --key-file /data/credentials.json
+        gcloud config set project openshift-gce-devel
+
```

Comment:
@stevekuznetsov would you be fine with this extension? Currently, it's impossible to set the original GCE SA and project i case the job fails before the artefacts are uploaded to the GCS bucket.

Reply:
Or maybe make it conditional in case someone really wants to access the GCS under different SA/project